development/compiling/compiling_for_web.rst

```diff
@@ -29,6 +29,11 @@ written by the Emscripten SDK, e.g. when invoking ``emsdk activate latest``,
 or by your package manager. It's also created when starting Emscripten's
 ``emcc`` program if the file doesn't exist.
 
+.. Attention::
```

Comment:
This should be `.. warning::`, and the paragraph should be indented. http://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#paragraph-level-markup

Reply:
Actually you're right, `.. attention::` is valid and defined in the sphinx_rtd theme: https://sphinx-rtd-theme.readthedocs.io/en/latest/demo/demo.html#admonitions
Should still be lowercase and indented for consistency.